SUMMARY.md

```diff
@@ -2,34 +2,61 @@
 
 - [Catalyst Product Management Curriculum](README.md)
 - [Facilitator Notes](mentor-notes.md)
-- [Supplementary Resources](resources.md)
+
+## Case Studies
+- [Programme Case Studies](case-studies/README.md)
+- [History of Tech for Better](case-studies/history.md)
+- [Tech vs. Abuse](case-studies/TvA/README.md)
+  - [TvA - Programme Structure](case-studies/TvA/TvA.md)
+- [My Best Life](case-studies/MBL/README.md)
+  - [MBL - Programme Structure](case-studies/MBL/MBL.md)
+- [Collaborative Digital Training](case-studies/CDT/README.md)
+  - [CDT - Programme Structure](case-studies/CDT/CDT.md)
+- [Discovery Learning Programme](case-studies/DLP/README.md)
+- [DLP - Programme Structure](case-studies/DLP/DLP.md)
+- [Tech for Better](case-studies/TFB/README.md)
+  - [T4B - Programme Structure](case-studies/TFB/TFB.md)
+
+## List of Exercises
+
+- [Exercises](list-of-exercises/README.md)
+- [The Five Whys](list-of-exercises/the-five-whys.md)
+- [The Bigger Picture](list-of-exercises/the-bigger-picture.md)
+- [Problem Statement](list-of-exercises/problem-statement.md)
+- [How Might We](list-of-exercises/how-might-we.md)
+- [Mapping](list-of-exercises/mapping.md)
+- [Assumptions Table](list-of-exercises/assumptions-table.md)
+- [Assumptions to Questions](list-of-exercises/assumptions-to-questions.md)
+- [Common Areas](list-of-exercises/common-areas.md)
+- [User Research Scripting](core-modules/discovery-workshop/user-research-scripting.md)
+- [Identifying Key Insights](core-modules/definition-workshop/identifying-key-insights.md)
+- [Inspiration](core-modules/definition-workshop/inspiration.md)
+- [Idea Generation](core-modules/definition-workshop/sketching/idea-generation.md)
+- [Creative 8s](core-modules/definition-workshop/sketching/creative-8s.md)
+- [Solution Sketch](core-modules/definition-workshop/sketching/solution-sketch.md)
+- [Dot Voting](core-modules/definition-workshop/sketching/dot-voting.md)
+- [User Stories](core-modules/definition-workshop/user-stories.md)
+- [User Stories Workshop](core-modules/definition-workshop/user-stories-workshop.md)
+- [User Journeys](core-modules/definition-workshop/user-journeys.md)
+- [User Testing](core-modules/definition-workshop/user-testing.md)
+
+## Extra resources
+
+- [Finding a Tech Partner](extra-resources/finding-a-tech-partner.md)
+- [Glossary](extra-resources/glossary.md)
+
+
+
+
+
 
 ## Core Modules
```

Comment:
Could we remove reference to core modules and pop all exercises in the list?

Reply:
All exercises are in the list but I didn't want to remove anything yet - will remove references in the summary though

Reply:
Ah, what I meant was moving User Personas etc. up into the main list - i.e. removing the headline "Core Modules" and "Discovery Workshop" but keeping the exercise itself.